README.md

```diff
@@ -1,13 +1,19 @@
 # Security Special Interest Group
 
-Covers horizontal security initiatives for the OpenTelemetry project, including regular security audits, the vulnerability management process, cross-cutting security documentation, and security community management.
+Covers horizontal security initiatives for the OpenTelemetry project, including
+regular security audits, the vulnerability management process, cross-cutting
+security documentation, and security community management.
 
-The [charter](https://docs.google.com/document/d/1fHuqKLeWwKYf5KBCpySqFTwCKuD2i8kK2D2RF3sVh4g/edit) draft defines the scope and governance of the Security Special Interest Group.
+The
+[charter](https://docs.google.com/document/d/1fHuqKLeWwKYf5KBCpySqFTwCKuD2i8kK2D2RF3sVh4g/edit)
```

Comment:
should this point to the charter in the repo?

Reply:
Yeah I’ll update it. Legacy from when I had a readme pr open. Was planning on just adding the charter subsequently

Reply:
Updated